### ISSUE TITLE
Run error when executing first time

### ISSUE THREAD

**Issue by mariohmol** [Urigo/meteor-angular-socially#120]
Hi!

I did a fork, ran `meteor`, it downloaded and installed, but after all it gives me a error.

Any ideas?

``````
meteor
[[[[[ ~/projetos/meteor-angular-socially ]]]]]

=> Started proxy.                             
=> Started MongoDB.                           
=> Meteor 1.3.2.4 is available. Update this project with 'meteor update'.

/Users/mariohmol/.meteor/packages/templating/.1.1.9.1h5n3gh++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:116
      throw error;
            ^
TypeError: Cannot call method 'split' of undefined
    at Object.parse (/tools/utils/parse-stack.js:18:28)
    at Object.exception (/tools/utils/buildmessage.js:497:29)
    at /tools/isobuild/compiler-plugin.js:189:26
    at /tools/utils/buildmessage.js:359:18
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:352:34
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:350:23
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.enterJob (/tools/utils/buildmessage.js:324:26)
    at /tools/isobuild/compiler-plugin.js:176:22
    at Function.time (/tools/tool-env/profile.js:305:10)
    at /tools/isobuild/compiler-plugin.js:175:15
    at Function._.each._.forEach (/Users/mariohmol/.meteor/packages/meteor-tool/.1.3.2.18upb5e++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at CompilerPluginProcessor.runCompilerPlugins (/tools/isobuild/compiler-plugin.js:165:7)
    at ClientTarget._runCompilerPlugins (/tools/isobuild/bundler.js:901:22)
    at /tools/isobuild/bundler.js:681:34
    at /tools/utils/buildmessage.js:359:18
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:352:34
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:350:23
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.enterJob (/tools/utils/buildmessage.js:324:26)
    at ClientTarget.make (/tools/isobuild/bundler.js:675:18)
    at /tools/isobuild/bundler.js:2474:14
    at /tools/isobuild/bundler.js:2563:20
    at Array.forEach (native)
    at Function._.each._.forEach (/Users/mariohmol/.meteor/packages/meteor-tool/.1.3.2.18upb5e++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:79:11)
    at /tools/isobuild/bundler.js:2562:7
    at /tools/utils/buildmessage.js:271:13
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:264:29
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:262:18
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:253:23
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.capture (/tools/utils/buildmessage.js:252:19)
    at Object.exports.bundle (/tools/isobuild/bundler.js:2455:31)
    at /tools/runners/run-app.js:591:36
    at Function.run (/tools/tool-env/profile.js:489:12)
    at bundleApp (/tools/runners/run-app.js:581:34)
    at AppRunner._runOnce (/tools/runners/run-app.js:634:35)
    at AppRunner._fiber (/tools/runners/run-app.js:887:28)
    at /tools/runners/run-app.js:411:12
``` `
``````

**Comment by netwiser** [comment]
Same issue here.
=> Started proxy.  
=> Meteor 1.3.4 is available. Update this project with 'meteor update'.

/root/.meteor/packages/templating/.1.1.9.155pxb4++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:116
      throw error;
            ^
TypeError: Cannot call method 'split' of undefined
    at Object.parse (/tools/utils/parse-stack.js:18:28)
    at Object.exception (/tools/utils/buildmessage.js:497:29)
    at /tools/isobuild/compiler-plugin.js:189:26
    at /tools/utils/buildmessage.js:359:18
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:352:34
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:350:23
....

**Comment by mariohmol** [comment]
I could make it run after running `meteor update` , but to make this update works i had to change /ets/hosts to include `54.192.225.217  warehouse.meteor.com` 

@netwiser , i really dont know how people are using meteor.. it looks so instable..

even after running meteor update, not it runs but i receive a lot of errors like:

```
imports/ui/components/auth/auth.ts (1, 26): Cannot find module 'angular'.
imports/ui/components/auth/auth.ts (2, 32): Cannot find module 'angular-meteor'.
imports/ui/components/auth/auth.ts (4, 24): Cannot find module 'meteor/meteor'.
imports/ui/components/auth/auth.ts (5, 26): Cannot find module 'meteor/accounts-base'.
imports/ui/components/auth/auth.ts (7, 27): Cannot find module './auth.html'.
imports/ui/components/auth/auth.ts (21, 10): Property 'helpers' does not exist on type 'Auth'.
imports/ui/components/login/login.ts (1, 26): Cannot find module 'angular'.
imports/ui/components/login/login.ts (2, 32): Cannot find module 'angular-meteor'.
imports/ui/components/login/login.ts (3, 27): Cannot find module 'angular-ui-router'.
imports/ui/components/login/login.ts (5, 24): Cannot find module 'meteor/meteor'.
imports/ui/components/login/login.ts (7, 30): Cannot find module './web.html'.
imports/ui/components/login/login.ts (8, 8): Module '"imports/ui/components/login/web"' has no default export.
imports/ui/components/login/login.ts (9, 33): Cannot find module './mobile.html'.
imports/ui/components/login/login.ts (10, 8): Module '"imports/ui/components/login/mobile"' has no default export.
imports/ui/components/login/mobile.ts (1, 26): Cannot find module 'meteor/accounts-base'.
imports/ui/components/login/mobile.ts (7, 10): Property '$state' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (11, 10): Property 'isStepTwo' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (12, 10): Property 'phoneNumber' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (13, 10): Property 'verificationCode' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (14, 10): Property 'error' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (18, 44): Property 'phoneNumber' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (18, 62): Property '$bindToContext' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (21, 14): Property 'error' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (23, 14): Property 'error' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (25, 14): Property 'isStepTwo' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (31, 31): Property 'phoneNumber' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (31, 49): Property 'verificationCode' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (31, 72): Property '$bindToContext' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (33, 14): Property 'error' does not exist on type 'Login'.
imports/ui/components/login/mobile.ts (36, 14): Property '$state' does not exist on type 'Login'.
```

**Comment by kamilkisiela** [comment]
I updated Socially so it's now using latest `urigo:static-templates` so it shouldn't be a problem now. But let's wait few hours for Uri to release new version of that package :)

@mariohmol About those errors, it's related to typescript. It's not an error by itself because files are still built. In the current version of the tutorial we don't learn how to use typescript, we just show how to migrate from angular1 with ecmascript to angular2 with typescript.

If you want to learn more about typescript, please visit [this chapter of Socially 2.0](http://www.angular-meteor.com/tutorials/socially/angular2/folder-structure)

**Comment by mariohmol** [comment]
@kamilkisiela thanks for asnwering..as it runs.. works.. but show erros.. anyway.. if just runnin meteor is not enought to make the app running should have somethin about how to setup in readme.. no?

**Comment by Urigo** [comment]
@mariohmol have you done `typings install` after?
Please also try from the new version that released today

**Comment by mariohmol** [comment]
HI!! the typings.json is missing.. even after running `typings install` and run `meteor`.. still with those errors..

what new version? looking at this repo the last commit is from feb/april

**Comment by Urigo** [comment]
https://github.com/Urigo/meteor-angular-socially was updated today

**Comment by mariohmol** [comment]
sorry.. u are right.. i saw last commit (april) ..  but files are from 1 day ago... 

i just cloned the new version.. ran.. the errors are the same.. what i did
- clone
- npm install
- meteor
- typings install (do not have typings.json but i ran anyway)

**Comment by mariohmol** [comment]
ps: as i'm still having problems.. why this issue was closed?

**Comment by mariohmol** [comment]
@netwiser could u fix it? is your enviroment giving same errors than mine?

**Comment by Urigo** [comment]
@mariohmol are those just Typescript errors? is the app and the import of html templates working?

If so, then it's a known issue that will be solved with Typescript 2.0 as described here: https://github.com/Urigo/angular2-meteor/tree/master/atmosphere-packages/css-compiler

**Comment by mariohmol** [comment]
to be honest i cant say if is working completly because i do not have other link to compare.. but looks like that is working. 

but good to know about angula2.. i will work in that version.. thanks!
